src/screens/discover/components/DiscoverScreenContent.js

```diff
@@ -20,7 +20,7 @@ function Switcher({ showSearch, children }) {
   );
 }
 
-export default function DiscoverSheetContent() {
+export default function DiscoverScreenContent() {
```

Comment:
This reused anywhere, or can it live in the `/screen` too?

Reply:
It is used in a bunch of places, but it is tightly related to the screen so I guess it makes sense to just import it from the screen